iocore/net/SSLNetVConnection.cc

```diff
@@ -2511,7 +2545,9 @@ SSLNetVConnection::update_early_data_config(uint32_t max_early_data, uint32_t re
 #else
     // If SSL_set_max_early_data is unavailable, it's probably BoringSSL,
     // and SSL_set_early_data_enabled should be available.
-    SSL_set_early_data_enabled(ssl, max_early_data > 0 ? 1 : 0);
+    bool const early_data_enabled = max_early_data > 0 ? 1 : 0;
+    SSL_set_early_data_enabled(ssl, early_data_enabled);
+    Debug("ssl", "Called SSL_set_early_data_enabled with %d", early_data_enabled);
     Warning("max_early_data is not used due to library limitations");
```

Comment:
@duke8253 : Should this warning be changed? Or removed? Maybe the intention here is that with boringssl there is no way to explicitly turn off replay (see line 2543 above).

Reply:
Yes, I think this can be removed here.

Reply:
Thanks. Removed.